src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -32,6 +32,7 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.CompleteStatement
     /// <summary>
     /// When user types <c>;</c> in a statement, semicolon is added and caret is placed after the semicolon
     /// </summary>
+    [Export]
```

Comment:
💭  This _shouldn't_ be necessary. I know I did work in this area in the past... checking...

Reply:
➡️ Yep, this is the extensions you want:

https://github.com/dotnet/roslyn/blob/afd10305a37c0ffb2cfb2c2d8446154c68cfa87a/src/EditorFeatures/TestUtilities/Extensions/ExportProviderExtensions.cs

Reply:
Resolved.